lib/account/account-manager.js

```diff
@@ -19,6 +36,28 @@
  * @typedef {Object} accountManager~confirmedInfo
  * @property {string} mfaKey
  * @property {string} passwordSalt
+ * @property {accountManager~hashSet} passwordHashing
+ */
+
+/**
+ * How to hash a password for registering and logging in.
+ *
+ * @typedef {Object} accountManager~hashSet
+ * @property {string} algo
+ * @property {string} hashLength
+ * @property {string} iterations
+ * @property {string} salt
+ */
+
+/**
+ * What an account holder needs to create the password hash
+ * to attempt to log in.
+ *
+ * @typedef {Object} accountManager~loginInitiateClientInfo
+ * @property {accountManager~hashSet} pbkdf
+ * @property {accountManager~challengeSet} challenge
+ * @property {string} encoding
+ *
```

Comment:
Would you add a property of `mfa` and set it to `["hotp"]`?  We don't need it but it also indicates that we require MFA for the account and what MFA is needed.  It's for the client's use *only*.